lib/pa11y.js

```diff
@@ -150,8 +150,12 @@ async function stateCleanup(state) {
 	if (state.browser && state.autoClose) {
 		await state.browser.close();
 	} else if (state.page) {
-		state.page.removeListener('request', state.requestInterceptCallback);
-		state.page.removeListener('console', state.consoleCallback);
+		try {
+			state.page.removeListener('request', state.requestInterceptCallback);
+			state.page.removeListener('console', state.consoleCallback);
+		} catch {
+			// do nothing
+		}
```

Comment:
Just need to provide an identifier for the caught error.  maybe log-debug, also?
```
} catch (error) {
  options.log.debug(error);
}
```

Reply:
`options` are not being passed in here at present... Would I need to pass them into the `stateCleanup` method?

```
async function stateCleanup(options, state) {
  ...
}

async function pa11y(url, options = {}, callback) {
  ...
  } finally {
    await stateCleanup(options, state);
  }
}
```

Reply:
Quite right, that would be enough.